README.md

```diff
@@ -1,34 +1,45 @@
-![logo](https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png)
-
-![CI](https://github.com/ManimCommunity/manim/workflows/CI/badge.svg)
-[![Documentation Status](https://readthedocs.org/projects/manimce/badge/?version=latest)](https://manimce.readthedocs.io/en/latest/?badge=latest)
-[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat)](http://choosealicense.com/licenses/mit/)
-[![Manim Subreddit](https://img.shields.io/reddit/subreddit-subscribers/manim.svg?color=ff4301&label=reddit)](https://www.reddit.com/r/manim/)
-[![Manim Discord](https://img.shields.io/discord/581738731934056449.svg?label=discord)](https://discord.gg/mMRrZQW)
-
-Manim is an animation engine for explanatory math videos. It's used to create precise animations programmatically, as seen in the videos at [3Blue1Brown](https://www.3blue1brown.com/).
-
-> NOTE: This repository is maintained by the Manim Community, and is not associated with Grant Sanderson or 3Blue1Brown in any way (though we are definitely indebted to him for providing his work to the world). If you want to study how Grant makes his videos, head over to his repository (3b1b/manim). This is a more frequently updated repository than that one, and is recommended if you want to use Manim for your own projects.
+<p align="center">
+    <img src="https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png" />
+    <br />
+    <br />
+    <img src="https://img.shields.io/badge/license-MIT-red.svg?style=flat" alt="MIT License" src="http://choosealicense.com/licenses/mit/" />
+    <img src="https://img.shields.io/reddit/subreddit-subscribers/manim.svg?color=orange&label=reddit" alt="Reddit" href="https://www.reddit.com/r/manim/" />
+    <img src="https://img.shields.io/discord/581738731934056449.svg?label=discord&color=yellow" alt="Discord" href="https://discord.gg/mMRrZQW" />
+    <img src="https://readthedocs.org/projects/manimce/badge/?version=latest" alt="Documentation Status" href="https://manimce.readthedocs.io/en/latest/?badge=latest" />
+    <img src="https://github.com/ManimCommunity/manim/workflows/CI/badge.svg" alt="CI" />
```

Comment:
I'd consider the fact that the CI is is passing and the docs are building to be more immediately useful to visitors than the license and social links, so I prefer the old order.

Additionally, it's odd for the reddit link to be orange rather than red and for the discord link to be yellow rather than green (which is a standard "working" color) or blue (which is part of discords branding). The change of the MIT button is a little better, but I don't really see any reason to change it from the standard blue used on shields.io.